obelisk/cpp/zoo/CMakeLists.txt

```diff
@@ -22,7 +22,20 @@ add_library(Zoo INTERFACE)
 add_library(Obelisk::Zoo ALIAS Zoo)
 
 target_include_directories(Zoo INTERFACE include)
-target_include_directories(Zoo INTERFACE robot)
+
+# conditionally add leap subdirectory if OBELISK_BUILD_LEAP is set
+if(OBELISK_BUILD_LEAP)
+  message(STATUS "Configuring Leap Hand Interface")
+  add_subdirectory(hardware/robots/leap)
+  target_include_directories(Zoo INTERFACE hardware/robots/leap)
+endif()
+
+# conditionally add zed subdirectory if OBELISK_BUILD_ZED is set
+if(OBELISK_BUILD_ZED)
```

Comment:
If these are environment variables, dont you need `$ENV{VAR}` to access them? https://cmake.org/cmake/help/latest/variable/ENV.html

Reply:
resolved: now, i also added an `OR` condition with the `$ENV` syntax, since per slack, these can be either a cmake arg or env var due to redundancies in obelisk, and I'm not sure how to properly distinguish between them.